pages/cart.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="javascriptreact">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	 <!-- CDN -->
+	 <link rel="stylesheet" href="https://vivid-ui.netlify.app/css/index.css" />
+	
+	 <link rel="stylesheet" href="/css/cart.css">
+	 <title>Cart | HitFit</title>
+</head>
+<body>
+	 <!-- navbar -->
+	 <nav class="navbar">
+		<div class="nav  d-flex flex-justify-between flex-items-center p-1">
+			<h1 class="brandname h3">HitFit</h1>
+			<div class="nav-items d-flex flex-items-center">
+				<div class="search d-flex">
+					<input type="text" class="search-box" placeholder="Search..">
```

Comment:
as an alternative, we can also give type="search" for search-related input. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search